src/main/java/com/rakovets/course/java/core/practice/oop/principles/hero/Warrior.java

```diff
@@ -0,0 +1,20 @@
+package com.rakovets.course.java.core.practice.oop.principles.hero;
+
+public class Warrior extends Hero{
+    private final int DAMAGE = 15;
+    private final int ARMOR_PERCENT = 30;
+    public Warrior(String name, int healthHero) {
+        super(name, healthHero);
+    }
+    public int getDAMAGE() {
+        return DAMAGE;
+    }
+
+    public void attackEnemy(Enemy enemy) {
+        System.out.println("The Warrior hero attacks");
+        enemy.takeDamage(DAMAGE);
+    }
+    public void tookDamage(Enemy enemy){
+        setHealthHero(getHealthHero()-(int)(enemy.getDamageEnemy() * ARMOR_PERCENT * 0.01f));
```

Comment:
"Violation: There isn't separating white spaces for operators
Solution: Add white spaces"